src/containers/VerletLists.h

```diff
@@ -147,34 +156,86 @@ class VerletLists : public LinkedCells<Particle, ParticleCell> {
   }
 
   bool isContainerUpdateNeeded() override {
-    for (int cellIndex1d = 0; cellIndex1d < this->_data.size(); ++cellIndex1d) {
-      std::array<double, 3> boxmin;
-      std::array<double, 3> boxmax;
+    for (unsigned long cellIndex1d = 0; cellIndex1d < this->_data.size(); ++cellIndex1d) {
```

Comment:
in LinkedCells.h size_t was used instead of unsigned long (yes I know its the same but some consistency would be nice)

Reply:
changed to size_t